service/src/test/org/apache/hive/service/cli/operation/TestOperationLogManager.java

```diff
@@ -61,6 +60,7 @@ public void setUp() throws Exception {
     HiveConf.setIntVar(hiveConf, HiveConf.ConfVars.HIVE_SERVER2_WEBUI_MAX_HISTORIC_QUERIES, 1);
     HiveConf.setIntVar(hiveConf, HiveConf.ConfVars.HIVE_SERVER2_WEBUI_PORT, 8080);
     HiveConf.setBoolVar(hiveConf, HiveConf.ConfVars.HIVE_IN_TEST, true);
+    HiveConf.setBoolVar(hiveConf, HiveConf.ConfVars.HIVE_TESTING_REMOVE_LOGS, false);
```

Comment:
why this property should set to false when HIVE_SERVER2_HISTORIC_OPERATION_LOG_ENABLED is enabled? we do not want to set this in production.

Reply:
This is because https://github.com/apache/hive/blob/master/ql/src/java/org/apache/hadoop/hive/ql/session/OperationLog.java#L80
    if (hiveConf.getBoolVar(HiveConf.ConfVars.HIVE_IN_TEST)) {
      isRemoveLogs = hiveConf.getBoolVar(HiveConf.ConfVars.HIVE_TESTING_REMOVE_LOGS);

Instead of changing the code there, I thought it may be better set it to false for this test case. In production no need to change this setting.